packages/stateless/src/lib/provider/provider.interface.ts

```diff
@@ -71,6 +71,10 @@ export enum EmailEventStatusEnum {
   BOUNCED = 'bounced',
   DROPPED = 'dropped',
   CLICKED = 'clicked',
+  BLOCKED = 'blocked',
+  SPAM = 'spam',
+  UNSUBSCRIBE = 'unsubscribe',
```

Comment:
Could you please follow the pattern of using the participle in both key and value?